### PR TITLE
docs: unify description for `suppress_embeds` across all files

### PR DESCRIPTION
**Pull request by onerandomusername**
## Summary

unify the description for the suppress_embeds parameter across all files and usages

## Checklist

- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: disnake/message.py [hunk 1]
```diff
@@ -1576,10 +1576,10 @@ async def edit(
                 Supports passing ``None`` to clear attachments.
 
         suppress_embeds: :class:`bool`
-            Whether to suppress embeds for the message. This removes
-            all the embeds if set to ``True``. If set to ``False``
-            this brings the embeds back if they were suppressed.
-            Using this parameter requires :attr:`~.Permissions.manage_messages`.
```
Comment (shiftinv) [inline disnake/message.py:1582]: The permission requirement should be kept here

Comment (onerandomusername) [reply disnake/message.py:1582]: https://github.com/DisnakeDev/disnake/pull/834/commits/fa6377dce151a49330d6c9042ea96f448a40176e

File: disnake/message.py [hunk 2]
```diff
@@ -1527,6 +1527,12 @@ async def edit(
             those images will be removed if the message's attachments are edited in any way
             (i.e. by setting ``file``/``files``/``attachments``, or adding an embed with local files).
 
+        .. warning::
+
+            This method cannot be used on messages authored by others, with one exception.
+            The ``suppress_embeds`` parameter can be provided for messages authored by others.
+            Changing ``suppress_embeds`` requires :attr:`~.Permissions.manage_messages`.
```
Comment (shiftinv) [inline disnake/message.py:1534]:
```suggestion
            This method cannot be used on messages authored by others, with one exception.
            The ``suppress_embeds`` parameter can be provided in that case,
            requiring :attr:`~.Permissions.manage_messages` permissions.
```
Perhaps something like this?

Comment (onerandomusername) [reply disnake/message.py]: I updated this somewhat in https://github.com/DisnakeDev/disnake/pull/834/commits/16e8d622f47ef749ee2b3a3c0222434a4b259bda

However, I've also realised that the `delete_after` parameter can be provided for messages authored by others, so I'm not sure of the best way to convey that.

Comment (shiftinv) [reply disnake/message.py]: Should be fine for now.

**Review by shiftinv — APPROVED** [review]

**Pull request merged**